lib/pbench/server/api/resources/__init__.py

```diff
@@ -343,9 +343,7 @@ def convert_list(value: List[Any], parameter: "Parameter") -> str:
         A new list with normalized elements
     """
     if type(value) is not list:
-        raise ConversionError(
-            str(value), f"List of {parameter.name}", type(value).__name__
-        )
+        raise ConversionError(value, f"List of {parameter.name}", type(value).__name__)
```

Comment:
I like that idea, although I'm not thrilled about the changes, either way. (Because if I can't remove it I'd want to make it a keyword argument for clarity, and either way I'll need to change every call site. Wonder how many there are? Well, OK, there are *eight*, and every one of them is `type(value).__name__`, which is stupid. So, yeah.)